images/js/googleCloud.js

```diff
@@ -0,0 +1,37 @@
+const { Storage } = require('@google-cloud/storage');
+
+const bucketName = 'gs://lottie-animation-community-tests';
```

Comment:
This should be async since you call it using 'await' above.

Reply:
fixed!